corehq/apps/style/static/style/lib/select2

```diff
@@ -1 +1 @@
-Subproject commit 6633cda9061d9fc65075dd140c63f9a6adbcbd72
+Subproject commit 3a10bd997a4d6a4a54e4bd4ebc210f51af0944b1
```

Comment:
No, I'm guessing it came in during a merge.  https://github.com/dimagi/commcare-hq/pull/6598